docs/blogs/IntroGatewayAPI/IntroGatewayAPI.md

```diff
@@ -0,0 +1,165 @@
+by [a large community](https://gateway-api.sigs.k8s.io/implementations/), including several vendors, proving that the API
+portability has significant appeal, enhances third-party integration, and reduces vendor lock-in.  GWAPI design is
+expressive, incorporating a native feature framework that replaces the use of custom annotations in Ingress.  At the same
+time, the design supports extensibility and the future of features through granular customization levels.  To simplify
+operations, GWAPI design focuses on role-oriented API resources, allowing for role-based access to just-enough Kubernetes
+details to satisfy and empower each organizational role.  Besides simplifying operations, role-oriented resource access
+reduces costs by enabling the safe sharing of infrastructure without requiring shared control.
+
+![Gateway API Role-Oriented Model](images/GatewayRoles.png)
+
+The key reason to use Gateway API with OpenShift Networking is that it enables the installation and use of multiple
+third-party GWAPI implementations on the same cluster.  This allows infrastructure providers to offer a range of gateway
+implementations for the benefit of cluster operators and application developers.  Also, as the standard implementations
+converge, the same GWAPI compliant configuration can be used for two different implementations on the same cluster.
+
+![Cluster with multiple Gateway API implementations](images/MultiGWAPI.png)
+
+### Features offered in Dev Preview
+
+In the Developer Preview we have chosen to provide the current version of GWAPI, 0.5.1, and the Istio release (1.15.1)
+that incorporates it. This release includes the following features:
+
+Feature | Definition
+-------------------|---------------------------------------------------------------------------------------------
+Path-based routing | Routes with different paths can have different destinations.|
+Weighted backends for traffic splitting | [Traffic Splitting]( https://gateway-api.sigs.k8s.io/guides/traffic-splitting/?h=weight#http-traffic-splitting) via weight configuration placed on backends, e.g. split 50% on each of two backends.
+Route sharding for isolation and performance | Use [GatewayAddress](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1alpha2.GatewayAddress) to bind Gateways to a specific shard.
+Wildcard subdomain | Host names may be prefixed with a wildcard label (*.), which is interpreted as a suffix match.
+Comprehensive route status | [Status overview](https://gateway-api.sigs.k8s.io/api-types/httproute/#routestatus) and [route conditions](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io%2fv1beta1.RouteConditionType) provide status and troubleshooting information on routes.
+Proxy logs | `oc logs pod frontend-POD-ID -c istio-proxy`
+Gateway logs | `oc logs pod gatewayname-POD-ID`
+Istiod logs | `oc logs pod istiod-POD-ID`
+Application logs| `oc logs pod app-name-POD-ID`
```

Comment:
The "pod" after "oc logs" is confusing because `oc logs` doesn't need (or accept) a resource/kind argument; "logs" should be followed by the pod name.  
```suggestion
Proxy logs | `oc logs <frontend-pod-name> -c istio-proxy`
Gateway logs | `oc logs <gateway-pod-name>`
Istiod logs | `oc logs <istiod-pod-name>`
Application logs| `oc logs <app-pod-name>`
```

Reply:
> I see some instances of "‘" ([U+2018](https://www.fileformat.info/info/unicode/char/2018/index.htm)) and "’" ([U+2019](https://www.fileformat.info/info/unicode/char/2019/index.htm)) instead of "'" ([U+0027](https://www.fileformat.info/info/unicode/char/0027/index.htm), a normal apostrophe). It's only in prose though, not in any code blocks, so it might be fine.

I used a backtick for code highlighting and code blocks.  If the original blog used single quotes anywhere, I didn't pick it up.